### ISSUE TITLE
tdp deployment fails at hdfs_audit_init node

### ISSUE THREAD

**Issue by GuillaumeHold** [TOSIT-IO/tdp-collection#675]
TDP deployment fails with a Mitogen error at the hdfs_audit_init node. The problem seems to involve TDP plugins of this collection.

Log :

```
TASK [tosit.tdp.common : Add directories for hdfs] *****************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ImportError: The Mitogen master process was unable to serve u'ansible_collections'. It may be a native Python extension, or it may be missing entirely. Check the importer debug logs on the master for more information.
failed: [edge-01] (item={'path': '/ranger/audit/hdfs', 'state': 'directory', 'owner': 'hdfs', 'group': 'hdfs', 'mode': '700'}) => changed=false 
  ansible_facts:
    discovered_interpreter_python: /usr/bin/python
  ansible_loop_var: item
  item:
    group: hdfs
    mode: '700'
    owner: hdfs
    path: /ranger/audit/hdfs
    state: directory
  module_stderr: |-
    Traceback (most recent call last):
      File "master:/home/guillaume/Documents/repos/tdp-getting-started/venv/lib/python3.10/site-packages/ansible_mitogen/runner.py", line 975, in _run
        self._run_code(code, mod)
      File "master:/home/guillaume/Documents/repos/tdp-getting-started/venv/lib/python3.10/site-packages/ansible_mitogen/runner.py", line 941, in _run_code
        exec('exec code in vars(mod)')
      File "<string>", line 1, in <module>
      File "master:/home/guillaume/Documents/repos/tdp-getting-started/ansible_collections/tosit/tdp/plugins/modules/hdfs_file.py", line 14, in <module>
      File "<stdin>", line 1490, in load_module
    ImportError: The Mitogen master process was unable to serve u'ansible_collections'. It may be a native Python extension, or it may be missing entirely. Check the importer debug logs on the master for more information.
  module_stdout: ''
  msg: |-
    MODULE FAILURE
    See stdout/stderr for the exact error
  rc: 1
```

**Comment by rpignolet** [comment]
Mitogen can not run task with custom modules so we must add `strategy: linear` into playbooks that launch tasks with custom modules like: https://github.com/TOSIT-IO/tdp-collection/blob/19cda90b97cdbc8d4d84cc7d52bca809a22914fb/playbooks/hive_hdfs_init.yml#L7

**Comment by GuillaumeHold** [comment]
Fixed by #679
